### DISCUSSION TITLE
Add some-sort of GUI?

### DISCUSSION THREAD

**Discussion by HalogenGam3r** [Inori/GPCS4#110] [Suggestions]
I just wanna to have a GUI w/o building the source code.
Probably isn't so simple to implement it.

**Comment by Inori** [comment]
You mean you wanna a GUI build tool?
Or GUI for the program?

**Reply by HalogenGam3r** [reply]
A GUI bulid, so I don't compile this bulid and debug it to run.

**Reply by VelocityRa** [reply]
What? You sound confused.
A graphical user interface, and automated builds/releases are 2 completely different things.

**Reply by HalogenGam3r** [reply]
I meant if **there's** a new bulid which is stable, then Inori should add GUI for easier accessing than building from source code.

**Reply by polybiusproxy** [reply]
That makes no sense. A GUI for the app itself would have, but a GUI just for accesing + building the source code? I've never seen that.

**Reply by HalogenGam3r** [reply]
It's like Kyty, who he had added a launcher to put the games and also homebrew.

**Reply by Inori** [reply]
I just want to focus on core functionality, others are less important as of now.
